### ISSUE TITLE
1.0.0 Bundle Size Large

### ISSUE THREAD

**Issue by bfaulk96** [axios/axios#5016]
#### Describe the issue
In `1.0.0`, npmjs.com reports an unpacked size of `1.3 MB`.
In the last release, which was `0.27.2`, the reported unpacked size is `446 kB`.

What has changed in 1.0.0 to almost double the number of files, and more than double the unpacked library size?

#### Example Code
N/A

#### Expected behavior, if applicable
1.0.0 switched to Rollup, which promised smaller bundling than WebPack. Perhaps there are some configurations missing?

#### Environment
 - Axios Version [1.0.0]
 - Node.js Version [v16.x.x]
 - OS: [OSX 12.6]

#### Additional context/Screenshots
The package size information used in this issue can be found [here (v1.0.0)](https://www.npmjs.com/package/axios) and [here (v0.27.2)](https://www.npmjs.com/package/axios/v/0.27.2)

**Comment by DigitalBrainJS** [comment]
Everything seems to be ok. Starting from v1.0.0, Axios switched to ESM modules, but for backward compatibility, we had to add two more prebuilt bundles to the `/dist` folder:
- browser ESM bundle
- node.js js bundle for use with require
It's just the package size, not the library size.

**Comment by jasonsaayman** [comment]
Yeah I think once actually bundled and minified in the app it should be normal, if not please let us know.

**Comment by bfaulk96** [comment]
Ah, that makes sense, sorry Jason!

**Comment by jasonsaayman** [comment]
No probs, if it was bigger that would definitely be something we would want to know about 😄

**Comment by guozhenyi** [comment]
It is acturally bigger and bigger. 😢 
![20240115134027](https://github.com/axios/axios/assets/5993771/15a9d128-441a-4a4b-9384-dec88e603423)

**Comment by suhaotian** [comment]
@guozhenyi  Do you want give a try [xior](https://github.com/suhaotian/xior) ? it's ~7kb around
